### PR TITLE
Gitter section

### PR DESCRIPTION
**Pull request by vasvi-sood**
gitter1
gitter2
gitter3

### CODE REVIEW

**Comment by abhishek-iiit** [comment]
Linked to issue #896

**Review by abhishek-iiit — CHANGES_REQUESTED** [review]
It would be nice for the user, if they find the `Blogs` section at the end. Could you please add as suggested and same for #1106 too Thank you:)

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by proudofsimin — APPROVED** [review]

**Comment by vasvi-sood** [comment]
> It would be nice for the user, if they find the `Blogs` section at the end. Could you please add as suggested and same for #1106 too Thank you:)

Changes done as suggested. Thankyou!

**Review by huan — APPROVED** [review]
LGTM.

```diff
- docusaurus/static/img/docs/howto/IM_platform/wechaty-puppet-gitter.webp 
+ docusaurus/static/img/docs/howto/deploy-gitter/wechaty-puppet-gitter.webp 
```

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Review by abhishek-iiit — APPROVED** [review]
LGTM

File: docusaurus/docs/howto/deploy-gitter.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+2. From the root directory install the necessary dependencies using the following commands:
+
+```bash
+npm install wechaty
+npm install wechaty-puppet-gitter
+```
+
+3. To pass the token, add the snippets below to the head of examples/ding-dong-bot.ts file:
+
+```bash
```
Comment (huan) [inline docusaurus/docs/howto/deploy-gitter.mdx]: `bash` -> `ts`

**Review by huan — APPROVED** [review]
Thanks for adding this gitter howto!

**Comment by Rohitesh-Kumar-Jain** [comment]
@vasvi-sood and @abhishek-iiit pls solve the merging conflict, and help yourself in merging the PR

File: docusaurus/docs/howto/deploy-gitter.mdx (hunk 1 above)
Comment (abhishek-iiit) [reply docusaurus/docs/howto/deploy-gitter.mdx]: Thankyou for your feedback, changes done as suggested :)

**Comment by abhishek-iiit** [comment]
Merging PR with no conflicts and 4 approvals :)

**Pull request merged**